test/populateTestData.js

```diff
@@ -0,0 +1,19 @@
+const { db, client } = require('../src/util/mongo');
+const createTestResource = async (data, resourceType) => {
+  const collection = db.collection(resourceType);
+  await collection.insertOne(data);
+  return { id: data.id };
+};
+//clean up db after test
+async function cleanUpDb() {
+  await db.dropDatabase();
+  await client.close();
+}
+
+const testSetup = async (pathToFiles, pathToPatients, pathToTestLibrary) => {
```

Comment:
Can we rename these arguments now that you updated it to pass in the json content rather than a path?

Reply:
I will make that change